core/templates/pages/topic-viewer-page/topic-viewer-page.component.html

```diff
@@ -102,8 +102,7 @@ <h2 class="coming-soon-title">Coming Soon!</h2>
 
   .oppia-topic-viewer-tabs-container {
     background-color: #00645C;
-    box-shadow: 3px 3px 6px rgba(0,0,0, 0.5);
```

Comment:
Please confirm regarding the box-shadow with Reshu.

Reply:
Hi @ReshuKumari, Please give your opinion on this.